docs/CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ nav_order: 6
 
     *Cameron Dutro*
 
+* Fix double rendering issue for partials that yield.
+
+    *Cameron Dutro*
```

Comment:
4.0.1 is already released, so I don't think this should be added as part of the 4.0.1 section

Reply:
Woops, thank you 👍